Gemfile.lock

```diff
@@ -383,7 +383,7 @@ GEM
     net-ssh (2.9.1)
     netrc (0.10.2)
     newrelic_rpm (3.9.9.275)
-    nio4r (1.0.1)
+    nio4r (1.1.1)
```

Comment:
Hi @marcaltmann, thanks for your note. I will upgrade the eye gem then. Shall I squash the commits also?